components/Header.tsx

```diff
@@ -144,6 +146,49 @@ export const Header = ({
                 {linkName}
               </Link>
             ))}
+            {drawerVisible ? (
+              <>
+                <Link
+                  href="https://www.simulationhockey.com"
+                  _hover={{ textDecoration: 'none' }}
+                  className={LINK_CLASSES}
+                >
+                  Forums
+                </Link>
+
+                <Link
+                  href="https://portal.simulationhockey.com"
+                  _hover={{ textDecoration: 'none' }}
+                  className={LINK_CLASSES}
+                >
+                  Portal
+                </Link>
+
+                <Link
+                  href="https://cards.simulationhockey.com"
+                  _hover={{ textDecoration: 'none' }}
+                  className={LINK_CLASSES}
+                >
+                  Cards
+                </Link>
+              </>
+            ) : (
+              <Menu>
+                <MenuButton className={LINK_CLASSES}>More</MenuButton>
```

Comment:
Doesn't really make sense to use the shared `LINK_CLASSES` constant for the menu button when the drawer isn't visible. We should define the styles so they match the other items in the header (for instance the hover state)